tutorials/vue/resize.md

```diff
@@ -0,0 +1,130 @@
+To create a resize event, you have a couple options.
+
+## Add an Event Listener
+
+This is the simpler approach of the two. You must utilize the
+`created()` and `destroyed()` lifecycle hooks to invoke the `addEventListener()`
+and `removeEventListener()` functions respectively.
+
+```javascript
+const app = new Vue({
+    data: function() {
+        return {
+            width: 0,
+            height: 0
+        }
+    },
+    created() {
+        window.addEventListener('resize', this.getDimensions);
+    },
+    destroyed() {
+        window.removeEventListener('resize', this.getDimensions);
+    },
+    methods: {
+        getDimensions() {
+            this.width = document.documentElement.clientWidth;
+            this.height = document.documentElement.clientHeight;
+        }
+    }
+})
+```
+
+The downside is that is would only be local to the component you are using it in. If you need something
```

Comment:
This sentence isn't great. Let's do "The downside is that any component that needs to listen to the 'resize' event needs to register global events. That's fine for a one-off, but can get messy if multiple components need to listen to the 'resize' event. Directives are an alternative approach that enable multiple components to listen to the 'resize' event without accessing `window`."